Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/calculation/DccWalletInfoCalculationManager.kt

```diff
@@ -62,17 +62,28 @@ class DccWalletInfoCalculationManager @Inject constructor(
 
     private suspend fun updateWalletInfoForPerson(person: PersonCertificates) {
         try {
-            val walletInfo = calculation.getDccWalletInfo(person.certificates)
+            val personIdentifier = person.personIdentifier ?: run {
```

Comment:
What about 
_person.personIdentifier.also{ checkNotNull(it) { "Person identifier is null. Cannot proceed." } }_

Reply:
I like `checkNotNull` here. 

I could simplify it to: 

```kotlin 
            val personIdentifier = checkNotNull(person.personIdentifier) { 
                "Person identifier is null. Cannot proceed." 
            }
```